gpMgmt/test/behave/mgmt_utils/gpactivatestandby.feature

```diff
@@ -76,6 +76,27 @@ Feature: gpactivatestandby
           And the tablespace is valid on the standby coordinator
           And clean up and revert back to original coordinator
 
+    Scenario: gprecoverseg recoveries failed segment when standby works instead of coordinator
+        Given the database is running
+          And all the segments are running
+          And the segments are synchronized
+          And the standby is not initialized
+         When the user initializes a standby on the host "sdw3" and port 7001
+         Then gpinitstandby should return a return code of 0
+          And verify the standby coordinator entries in catalog
+          And verify that pg_hba.conf file has "standby" entries in each segment (primary and mirror) data directories
+          And the coordinator goes down
+          And the user runs gpactivatestandby with options "-f"
+         Then gpactivatestandby should return a return code of 0
+          And verify the standby coordinator is now acting as coordinator
+          And user restarts from standby all primary processes for content 0
```

Comment:
Maybe use an already existing step `user restarts from standby all {segment_type} processes for content {content}`?

Reply:
> step `user restarts from standby all {segment_type} processes for content {content}`

was added by this patch

Reply:
It will work, if standby works in the same host as coordinator, but this test runs standby at another host.